test/mergeSort.js

```diff
@@ -0,0 +1,27 @@
+import { expect } from 'chai'
+import mergeSort from '../src/mergeSort'
+
+describe('mergeSort()', function(){
+  let arr
+  beforeEach(function() {
+    arr = new Array();
+    arr.push(34)
+    arr.push(203)
+    arr.push(3)
+    arr.push(746)
+    arr.push(200)
+    arr.push(984)
+    arr.push(198)
+    arr.push(764)
+    arr.push(9)
```

Comment:
You shouldn't have to build an unsorted array here to test.  I think you can just give an unsorted array to perform ```mergeSort()``` on

```
const unsortedArray = [ 34, 203, 3, 746, 200, 984, 198, 764, 9]
```